cmd/query/app/querysvc/adjuster/clockskew.go

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package adjuster
+
+import (
+	"encoding/binary"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/jaegertracing/jaeger/internal/jptrace"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+const (
+	warningDuplicateSpanID       = "duplicate span IDs; skipping clock skew adjustment"
+	warningFormatInvalidParentID = "invalid parent span IDs=%s; skipping clock skew adjustment"
```

Comment:
Why does this have format in the name? Are we warning about id format? I'd also tweak the error to be more specific - invalid is not really descriptive. I think here we mean that parent span id is not in the trace.

Reply:
@yurishkuro I think the "format" is there is because this is a format string. I removed the word format and changed the warning string per your suggestion.